acceptance-tests/src/test-support/java/tech/pegasys/artemis/test/acceptance/dsl/ArtemisNode.java

```diff
@@ -137,6 +168,9 @@ public Config() {
 
       final Map<String, Object> beaconRestApi = getSection(BEACONRESTAPI_SECTION);
       beaconRestApi.put("portNumber", REST_API_PORT);
+
+      final Map<String, Object> database = getSection(DATABASE_SECTION);
+      database.put("startFromDisk", false);
```

Comment:
This is the default anyway so we can just leave it unset in the constructor.  startFromDisk will still set it correctly when needed.

Reply:
done.